packages/rrweb-snapshot/typings/types.d.ts

```diff
@@ -6,50 +6,50 @@ export declare enum NodeType {
     CDATA = 4,
     Comment = 5
 }
-export declare type documentNode = {
+export type documentNode = {
```

Comment:
No action required, just curious: Why remove the `declare` here?

Reply:
I think this was auto generated? I'll undo and see what happens

Reply:
Yeah not sure what happened before, but `export type` is the correct statement (it also watches others in the file for `type`).